api/schema/schema_validator.go

```diff
@@ -24,11 +24,12 @@ func NewSchemaValidator(t *testing.T, schemaName string, got interface{}) *Schem
 	if err != nil {
 		t.Fatal(err)
 	}
+	// Platform compatibility: use "/" separators always for file://
+	dir = strings.Replace(dir, "\\", "/", -1)
```

Comment:
I don't think we should do that. Let's not introduce additional complexity. I am all for the comment that I [originally suggested](https://github.com/github/git-lfs/pull/1374/files/ee5df4971f3a819607d31ce91841417141ae1b84#r70994673).